Live/LiveEventWithDVR/Program.cs

```diff
@@ -182,7 +179,7 @@ private static async Task RunAsync(ConfigWrapper config)
 
                 // Create the LiveEvent Preview IP access control object. 
                 // This will restrict which clients can view the preview endpoint
-                LiveEventPreview liveEventPreview = new LiveEventPreview
+                LiveEventPreview liveEventPreview = new()
```

Comment:
Ok, getting used to this syntax now... fine with it.

Reply:
👍 Like me